jena-extras/jena-commonsrdf/README.md

```diff
@@ -0,0 +1,14 @@
+# commonsrdf-jena
+
+Code showing using [Apache CommonsRDF](http://commonsrdf.incubator.apache.org/)
```

Comment:
I believe that the correct URL is https://commons.apache.org/proper/commons-rdf/ (commons RDF is no longer in the incubator)

Reply:
That's a bit of a give away about the original timeline of the code!

(The code has also been de-java11'ed.)